hax-lib-protocol/src/lib.rs

```diff
@@ -1,14 +1,11 @@
-//! This crate provides types and traits for implementing a protocol state
-//! machine.
-//!
-//! A protocol party is conceived of as having a set of possible states, one of
-//! which is the initial state. Transitioning to a different state is possible
-//! either through receiving and processing a message or through writing a
-//! message.
+pub mod cal;
```

Comment:
What does `cal` mean?

Can you add new crate docs?

Reply:
`cal` could be short for cryptographic abstraction layer, but yeah it pretty opaque! How about `abstract_crypto` instead?

Reply:
Right now it's not very abstract 🙂 I'd probably just call it `crypto`. But you can as well keep `cal` and just put a doc comment on it to explain what it means.

Reply:
That's true. :D
I like your suggestion.